spec/spec_helper.rb

```diff
@@ -13,4 +15,16 @@
   config.expect_with :rspec do |c|
     c.syntax = :expect
   end
+
+  VCR.configure do |config|
+    config.cassette_library_dir = 'spec/fixtures/vcr_cassettes'
+    config.hook_into :webmock
+    config.configure_rspec_metadata!
+  end
```

Comment:
```suggestion
  VCR.configure do |vcr_config|
    vcr_config.cassette_library_dir = 'spec/fixtures/vcr_cassettes'
    vcr_config.hook_into :webmock
    vcr_config.configure_rspec_metadata!
  end
```

to avoid confusion with the `Rspec config`. -- rubocop complained about this.